src/KeyboardAccessoryView.tsx

```diff
@@ -13,6 +13,7 @@ interface Props {
   panResponderPositionY?: Animated.Value
   spaceBetweenKeyboardAndAccessoryView?: number
   style?: StyleProp<ViewStyle>
+  renderBackgroundElement?: () => React.ReactNode
```

Comment:
Alphabetic order, rename to `renderBackgroundNode` since we expect a node, not element.